advocacy_docs/security/advisories/index.mdx

```diff
@@ -135,76 +159,91 @@ An issue was discovered in EnterpriseDB Postgres Advanced Server (EPAS) before 1
 <hr/>
 <em>Summary:</em>&nbsp;
-A vulnerability was found in PostgreSQL versions 11.x prior to 11.3. The Windows installer for EDB-supplied PostgreSQL does not lock down the ACL of the binary installation directory or the ACL of the data directory; it keeps the inherited ACL. In the default configuration, this allows a local attacker to read arbitrary data directory files, essentially bypassing database-imposed read access limitations. In plausible non-default configurations, an attacker having both an unprivileged Windows account and an unprivileged PostgreSQL account can cause the PostgreSQL service account to execute arbitrary code.
+A vulnerability was found in PostgreSQL versions 11.x prior to 11.3. The Windows installer for EnterpriseDB-supplied PostgreSQL does not lock down the ACL of the binary installation directory or the ACL of the data directory; it keeps the inherited ACL. In the default configuration, this allows a local attacker to read arbitrary data directory files, essentially bypassing database-imposed read access limitations. In plausible non-default configurations, an attacker having both an unprivileged Windows account and an unprivileged PostgreSQL account can cause the PostgreSQL service account to execute arbitrary code.
 <br/>
 <a href="cve201910128">Read More...</a>
 </details></td></tr>
-</table>
 
-## Updated 2018
 
-<table>
+
+
+</table>   
+<h2>Updated 2018</h2>
+
+<table class="table-bordered">
+
+
 <tr><td>
-<details><summary><h3 style="display:inline">CVE-2007-4639</h3>
+<details><summary><h3 style="display:inline"> CVE-2007-4639 </h3>
 <span>
 &nbsp;&nbsp;<a href="cve20074639">Read Advisory</a>
-&nbsp;&nbsp;Updated: </span><span>2018/10/15</span>
-<h4>EDB Advanced Server 8.2 improperly handles debugging function calls
-</h4>
-<h5>EDB Postgres Advanced Server version 8.2</h5>
+&nbsp;&nbsp;Updated: </span><span>2018/10/15 </span>
+<h4>EDB Advanced Server 8.2 improperly handles debugging function calls</h4>
+<h5> EDB Advanced Server 8.2</h5>
 </summary>
 <hr/>
 <em>Summary:</em>&nbsp;
-EDB Postgres Advanced Server 8.2 does not properly handle certain debugging function calls that occur before a call to <code>pldbg_create_listener</code>, which allows remote authenticated users to cause a denial of service (daemon crash) and possibly execute arbitrary code via a SELECT statement that invokes a <code>pldbg_</code> function, as demonstrated by (1) <code>pldbg_get_stack</code> and (2) <code>pldbg_abort_target</code>, which triggers use of an uninitialized pointer.<br/>
+EDB Postgres Advanced Server 8.2 (EPAS) does not properly handle certain debugging function calls that occur before a call to `pldbg_create_listener`, which allows remote authenticated users to cause a denial of service (daemon crash) and possibly execute arbitrary code via a SELECT statement that invokes a `pldbg_` function, as demonstrated by (1) `pldbg_get_stack` and (2) `pldbg_abort_target`, which triggers use of an uninitialized pointer.
```

Comment:
Oh... This snuck through (Markdown code formatting in HTML block)

Reply:
Gah yes, will have to HTMLise the summary at least.

Reply:
Added to templates a cleaning regexp to switch backticks to code. Should be sufficient for now.